File: app/src/main/java/com/sermage/mymoviecollection/pojo/TrailerResponse.java
package com.sermage.mymoviecollection.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class TrailerResponse {
    @SerializedName("id")
    @Expose
    private int id;
    @SerializedName("results")
    @Expose
    private List<Trailers> trailers = null;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public List<Trailers> getTrailers() {
        return trailers;
    }

    public void setTrailers(List<Trailers> trailers) {
        this.trailers = trailers;
    }
}

File: app/src/main/java/com/sermage/mymoviecollection/pojo/TrailerResponse.kt
package com.sermage.mymoviecollection.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TrailerResponse (
        @SerializedName("id")
    @Expose
    val id: Int = 0,

        @SerializedName("results")
    @Expose
    val trailers: List<Trailers>? = null
)